### ISSUE TITLE
Indent style and size are ignored for default (`*`) wildcard

### ISSUE THREAD

**Issue by atipugin** [editorconfig/editorconfig-vscode#343]
Please fill-in this template.

- [x] I have a question that is specific to this extension; thus, inappropriate for the main [EditorConfig issue tracker](https://github.com/editorconfig/editorconfig/issues).
- [x] I tried running `code --disable-extensions` and the issue did **NOT** present itself.

Delete the following condition if it doesn't apply to your case:

If the extension is not picking up the expected configuration for a file:
- [x] I tried `npm install editorconfig -g` and ran `editorconfig [file-in-question]` and the configuration was what I expected. If not, please file on the [`editorconfig-core-js` issue tracker](https://github.com/editorconfig/editorconfig-core-js/issues).

## Issue

|             | Visual Studio Code | editorconfig-vscode |
|-------------|--------------------|---------------------|
| **Version** | `1.74.0`            | `0.16.4`             |

### Root `.editorconfig` File

```ini
root = true

[*]
charset = utf-8
end_of_line = lf
indent_size = 2
indent_style = space
insert_final_newline = true
trim_trailing_whitespace = true
```

Are there any other relevant `.editorconfig` files in your project? Yes / No

| Visual Studio Code Setting     | Default | User    | Workspace |
|--------------------------------|---------|---------|-----------|
| `editor.insertSpaces`          | `true`  | `____`  | `____`    |
| `editor.tabSize`               | `4`     | `_`     | `_`       |
| `editor.trimAutoWhitespace`    | `true`  | `____`  | `____`    |
| `files.autoSave`               | `"off"` | `"___"` | `"___"`   |
| `files.insertFinalNewline`     | `false` | `_____` | `_____`   |
| `files.trimTrailingWhitespace` | `false` | `_____` | `_____`   |

### File opened

`./hello.md`

### Expected behavior

```ini
indent_size = 2
```

### Actual behavior

```ini
indent_size = 4
```

Actually VSCode says following in status bar: `Spaces: 4 (Tab Size: 2)`.

### Additional comments or steps to reproduce

Tried to disable `editor.detectIndentation` - still no luck, VSCode keeps ignoring `[*]` settings and uses 4 spaces.

**Comment by rspcactucker** [comment]
I have this issue as well.

Here are the settings via CLI `editorconfig` :

```bash
$ editorconfig app/client/src/bundles/bundle.js 
charset=utf-8
end_of_line=lf
indent_size=2
indent_style=space
insert_final_newline=true
trim_trailing_whitespace=false
tab_width=2
```

And here is what VS Code is reporting:

![image](https://user-images.githubusercontent.com/54301657/207237710-6ec001e2-709e-4e6b-bd6d-17af0a614c1f.png)

For reference, here is my `.editorconfig` file:

```editorconfig
root = true

[*]
charset = utf-8
end_of_line = lf
indent_size = 2
indent_style = space
insert_final_newline = true
trim_trailing_whitespace = false

[*.{json,php}]
indent_size = 4
indent_style = space
```

**Comment by atipugin** [comment]
I think it might be related to #341 and https://github.com/microsoft/vscode/issues/168836

**Comment by Macil** [comment]
Yeah, I'm seeing this too. I have a [project](https://github.com/Macil/aocd) that has no .vscode indentation settings, and an [.editorconfig](https://github.com/Macil/aocd/blob/main/.editorconfig) specifying `indent_style = space` `indent_size = 2` for all `*` files.

Previously, if you made a new file in the project, VSCode would show "Spaces: 2" in the status bar. Now it shows "Spaces: 4 (Tab Size: 2)", and indents to 4 spaces on tab key press. (If I remove the .editorconfig file, then it shows "Spaces: 4" instead. This shows that the .editorconfig is being recognized but isn't having the proper effect.)

**Comment by atipugin** [comment]
VSCode team have just released 1.74.1 update and now problem is gone 🎉 Closing the issue.
